docs/requirement-sets/requirement-sets.md

```diff
@@ -19,16 +19,16 @@ To work as expected, your Office Add-in might depend on a specific Office applic
+|Office 2021 on Mac<br>(one-time purchase)|- TaskPane<br>- Content<br>- [CustomFunctions](excel/custom-functions-requirement-sets.md)<br>- [Add-in Commands](common/add-in-commands-requirement-sets.md)|- [ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)<br>- [ExcelApi 1.2](excel/excel-api-1-2-requirement-set.md)<br>- [ExcelApi 1.3](excel/excel-api-1-3-requirement-set.md)<br>- [ExcelApi 1.4](excel/excel-api-1-4-requirement-set.md)<br>- [ExcelApi 1.5](excel/excel-api-1-5-requirement-set.md)<br>- [ExcelApi 1.6](excel/excel-api-1-6-requirement-set.md)<br>- [ExcelApi 1.7](excel/excel-api-1-7-requirement-set.md)<br>- [ExcelApi 1.8](excel/excel-api-1-8-requirement-set.md)<br>- [ExcelApi 1.9](excel/excel-api-1-9-requirement-set.md)<br>- [ExcelApi 1.10](excel/excel-api-1-10-requirement-set.md)<br>- [ExcelApi 1.11](excel/excel-api-1-11-requirement-set.md)<br>- [ExcelApi 1.12](excel/excel-api-1-12-requirement-set.md)<br>- [ExcelApi 1.13](excel/excel-api-1-13-requirement-set.md)<br>- [ExcelApi 1.14](excel/excel-api-1-14-requirement-set.md)<br>- [ExcelApi 1.15](excel/excel-api-1-15-requirement-set.md)<br>- [DialogApi 1.1](common/dialog-api-requirement-sets.md)<br>- [IdentityAPI 1.3](common/identity-api-requirement-sets.md)<br>- [ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)<br>- [ImageCoercion 1.2](common/image-coercion-requirement-sets.md#imagecoercion-12)<br>- [OpenBrowserWindowApi 1.1](common/open-browser-window-api-requirement-sets.md)<br>- [RibbonApi 1.1](common/ribbon-api-requirement-sets.md)<br>- [SharedRuntime 1.1](common/shared-runtime-requirement-sets.md)<br>- [KeyboardShortcuts 1.1](common/keyboard-shortcuts-requirement-sets.md)|- [BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)<br>- [CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)<br>- [DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)<br>- [File](common/office-add-in-requirement-sets.md#file)<br>- [MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)<br>- [MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)<br>- [PdfFile](common/office-add-in-requirement-sets.md#pdffile)<br>- [Selection](common/office-add-in-requirement-sets.md#selection)<br>- [Settings](common/office-add-in-requirement-sets.md#settings)<br>- [TableBindings](common/office-add-in-requirement-sets.md#tablebindings)<br>- [TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)<br>- [TextBindings](common/office-add-in-requirement-sets.md#textbindings)<br>- [TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)|
```

Comment:
The 2019 Mac section has asterisks by later requirement sets. Should 1.15 have that here too?